components/Common/AvatarGroup/index.tsx

```diff
@@ -0,0 +1,62 @@
+'use client';
+import * as Avatar from '@radix-ui/react-avatar';
+import { useState } from 'react';
+import type { FC } from 'react';
+import styles from './index.module.css';
+
+type AvatarGroupProps = {
+  avatars: {
+    src: string;
+    name: string;
+  }[];
+  limit?: number;
+};
+
+const userNameToFallBackText = (name: string) => {
+  // sanitize name before using it
+  name = name.toLowerCase().replace(' ', '-');
```

Comment:
It is discouraged to reassign/mutate any value that comes from an argument.

Reply:
so create a new variable call result.